src/sql/parts/notebook/cellViews/outputArea.component.ts

```diff
@@ -34,7 +34,9 @@ export class OutputAreaComponent extends AngularDisposable implements OnInit {
 		this.updateTheme(this.themeService.getColorTheme());
 		if (this.cellModel) {
 			this._register(this.cellModel.onOutputsChanged(() => {
-				this._changeRef.detectChanges();
+				if (!(this._changeRef['destroyed'])) {
```

Comment:
Should we add the same condition on all the cells wherever detect change happens?

Reply:
We could, but this is the only place that I've seen where this error can occur, since we're reusing the output component. I'm a little worried about the code component and checking this condition thousands of times unnecessarily.